src/app/containers/PageHandlers/withContexts.jsx

```diff
@@ -27,8 +29,10 @@ const WithContexts = Component => {
       pageData,
     } = props;
 
+    const remoteToggles = pathOr(null, ['toggles'], remoteConfig);
```

Comment:
Should this default to null? Does not mean that if remoteConfig is not available the no toggles/confit are set? Do we not want to use the default feature toggles currently in Simorgh?

Reply:
As is, these get passed into the ToggleContext, and get merged into the default toggles in there. It might be easier to understand what's going on if we move that to the `getRemoteConfig` function, and just pass all the toggles in from the top of the app? We'd leave the ToggleContext as is then and just let it accept the initial toggles as a prop.

Reply:
That way we can just use those toggle values on the server too. We were chatting about this a bit yesterday!

Reply:
In the swarm review, it was decided that the work to merge the remote toggles with the default toggles with the default function written to do the fetching of the toggles in this PR. This can then be passed down to avoid having to replace any `null` values we might currently see.